src/batou_scm/tests/test_source.py

```diff
@@ -7,14 +7,13 @@
 
 @pytest.fixture
 def source(root):
-    source = Source(
-        dist_sources=repr([
-            'hg+https://example.com/foo',
-            'hg+https://example.com/bar branch=BAR',
-        ]),
-        sources=repr([
-            'hg+https://example.com/baz revision=BAZ',
-        ]))
+    source = Source(dist_sources=repr([
```

Comment:
Why not keep `dist_sources` on the next line. Now the indentation looks odd because more inner parts are less indented than the outer ones (aka `souces`).

Reply:
2to3 i guess